.env.development.example

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://localhost/bix_dev
```

Comment:
My local dev environment was a little different and I didn't want to commit my db config. Thought I'd move the file out of the way and add a note in the README